bot/cogs/bot.py

```diff
@@ -60,6 +62,18 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        """
+        Auto codeblock python code
+        """
+        if msg.content.count("\n") >= 3:  # more than three lines
+            try:
+                tree = ast.parse(msg.content)  # no syntax errors
+                if not all(isinstance(node, ast.Expr) for node in tree.body):  # we dont want hi\nthere\nguys\nD
```

Comment:
This inline comment seems unnecessary, maybe have a comment underneath describing the check more clearly?

Reply:
Inline comments never really look nice, and a couple here are probably un-needed, especially after the if statement and the tree definition. Moving them above or underneath a line, and adding newlines around that is a better idea.

Reply:
alright

Reply:
Your comments should follow PEP8 - they should start with a capital letter, and need to be grammatically correct. That means you're gonna need to use some punctuation. This comment, for example, deseperately needs a period and a comma.